index.html

```diff
@@ -27,6 +27,7 @@
 </head>
 
 <body data-title="index" id="body" style="background: #96D3F3;">
+
```

Comment:
Unneeded new empty line (doesn't improve code readability). It breaks the file's convention, too.